framework/update/helper/platform_mac.cpp

```
@@ -0,0 +1,87 @@
+    while (waited < timeoutMs) {
+        if (::kill(static_cast<pid_t>(pid), 0) != 0) {
+            // No such process -> it has exited.
+            return true;
+        }
+        sleepMs(step);
+        waited += step;
+    }
+    return ::kill(static_cast<pid_t>(pid), 0) != 0;
+}
+
+bool verifyInstall(const std::string& path)
```

Cppcheck (2.21.0)

[style] The function 'verifyInstall' is never used.

(unusedFunction)